spec/js_routes/options_spec.rb

```diff
@@ -13,6 +13,21 @@
   let(:_options) { {} }
   let(:_warnings) { true }
 
+  context "when serializer is specified" do
+    let(:_options) { {:serializer => "myCustomSerializer"} }
+
+    it "should set configurable serializer" do
+      # define custom serializer
+      # this is a nonsense serializer, which always returns foo=bar
+      # for all inputs
+      evaljs(%q(function myCustomSerializer(object, prefix) { return "foo=bar"; }))
+
+      # expect the nonsense serializer above to have appened foo=bar
+      # to the end of the path
+      expect(evaljs(%q(Routes.inboxes_path()))).to include("?foo=bar")
```

Comment:
@bogdan I did this because this particular test isn't concerned with the rest of the output of `Routes.inboxes_path` the `/inboxes` part should be tested by some other test somewhere else. This test is only concerned with the `?foo=bar` portion of the output.

If you really prefer it, I can switch the test to the code below, but I wrote it the other way on purpose.

    expect(evaljs(%q(Routes.inboxes_path()))).to eql("/inboxes?foo=bar")